### ISSUE TITLE
Zimfarm workers: can we have two workers behind the same public IP?

### ISSUE THREAD

**Issue by benoit74** [kiwix/operations#323]
We now have two Zimfarm workers `rhys` and `kathrin` with both the same public IP (they are both operated by same person). 

I think this poses a challenge in terms of respecting the constraints we have on platforms. E.g. wikihow where we do not want more than one task at a time per public IP, but currently the constraint is expressed in terms of one task per worker.

Is there other problems? WDYT?

**Comment by rgaudin** [comment]
I think we should not allow multiple workers per public IP until this has been clarified.

I don't think the documentation needs to be updated at this stage.

**Comment by benoit74** [comment]
I agree this looks like the simplest solution, I'm concerned we might miss something by moving too fast on this (and it is not like we have a huge benefit of supporting multiple workers behind one public IP).
